ospf6d/ospf6_abr.c

```diff
@@ -1194,9 +1194,10 @@ void ospf6_abr_examin_summary(struct ospf6_lsa *lsa, struct ospf6_area *oa)
 				   __func__, &prefix, listcount(old->paths));
 	}
 	for (old_route = old; old_route; old_route = old_route->next) {
-		if (!ospf6_route_is_same(old_route, route) ||
-			(old_route->type != route->type) ||
-			(old_route->path.type != route->path.type))
+		if (!ospf6_route_is_same(old_route, route))
+			break;
```

Comment:
would it make sense to put a comment here just so folks looking at this in the future understand why this break is here? it's fine if the answer is "no," just thought I'd ask ... :-)

Reply:
will do this - thankds @riw777